pkg/cli/cmd/radinit/aws.go

```diff
@@ -31,48 +31,94 @@ const (
 	QueryRegion = "us-east-1"
 
 	selectAWSRegionPrompt                 = "Select the region you would like to deploy AWS resources to:"
+	selectAwsCredentialKindPrompt         = "Select a credential kind for the AWS credential:"
```

Comment:
@willtsai @Reshrahim  cc-ing for reviewing all UX text.

Reply:
Synced with @vishwahiremat on the UX prompts.

Reply:
added the fix as discussed